dev/TEAL_opcodes.md

```diff
@@ -78,7 +78,7 @@ Ops have a 'cost' of 1 unless otherwise specified.
 - Pushes: uint64
 - A times B. Panic on overflow.
 
-It is worth noting that there are 10,000,000,000,000,000 micro-Algos in the total supply, or a bit less than 2^54. When doing rational math, e.g. (A * (N/D)) as ((A * N) / D) one should limit the numerator to less than 2^10 to be completely sure there won't be overflow.
+It is worth noting that there are 10,000,000,000,000,000 micro-Algos in the total supply, or a bit less than 2^54. When doing rational math, e.g. (A * (N/D)) as ((A * N) / D) one should limit the numerator to less than 2^10 to be completely sure there won't be overflow. Overflow is an error condition which halts execution and fails the transaction. Full precision is available from `mullw`.
```

Comment:
Add more context to what `A`, `N`, and `D` are supposed to represent.

For example:
 For example, let us assume that `A` is a transaction amount and `N` and `D` are the numerator and the denominator of some rational number. Then if (A*N) / D needs to be computed, `N` needs to be less than 2^10 to be completely sure there will be no overflow.

Reply:
This whole paragraph is non-normative nice-to-have text, it doesn't describe any requirement of the system, it's just a helpful reminder. I don't want to make it any bigger. If it's distracting for not being fully precise I'd rather remove it entirely and let people educate themselves on how math on binary computers works.